src/lib/core/a11y/list-key-manager.ts

```diff
@@ -32,10 +32,18 @@ export class ListKeyManager {
     return this;
   }
 
-  /** Sets the focus of the list to the item at the index specified. */
-  setFocus(index: number): void {
+  /**
+   * Sets the focus of the list to the item at the index specified.
+   *
+   * @param index The index of the item to be focused.
+   * @param focusElement Whether or not to focus the element as well. Defaults to `true`.
+   */
+  setFocus(index: number, focusElement = true): void {
```

Comment:
Actually, `focus` should be a no-op if the item is already focused. Why not just let it run again?